oak_functions/loader/src/tests.rs

```diff
@@ -413,3 +408,10 @@ async fn test_apply_policy() {
         assert_eq!(response.body.len(), policy.constant_response_size_bytes);
     }
 }
+
+#[test]
+fn test_try_from_utf8() {
```

Comment:
```suggestion
fn test_try_from_utf8_with_incorrect_utf8_input() {
```
supernit!

Reply:
Well, it's testing both correct and incorrect UTF8, right? I don't think it's worth separating into different functions, is it?

Reply:
Sorry, I misread the behaviour of `try_from_utf8`. This looks good.